multiple_generate.sh

```diff
@@ -7,19 +7,18 @@ function clean() {
 function cloner() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  git clone $1 ${REPONAME}_temp &
+  git clone $1 ${REPONAME} &
 }
 
-function repoCleaner() {
+function genSubIndex() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  cd ${REPONAME}_temp
-  mkdir ../$REPONAME
-  cp -a docs/. ../$REPONAME/.
-  cd ..
-  rm -rf ${REPONAME}_temp
+  cd $REPONAME/docs
+  python $BASE/modules/scripts/genindex.py ./
```

Comment:
genindex.py searches for README in the the path passed to it. README won't be inside REPONAME/docs. It would be in REPONAME/